docs/samples/git-setup/policytemplates/version_4.Y.Z/kustomization.yaml

```diff
@@ -2,7 +2,8 @@ generators:
 - sno-ran-du/sno-ran-du-pg-v4-Y-Z-v1.yaml
 # This ACM PG is needed when the previous one has to be updated.
 - sno-ran-du/sno-ran-du-pg-v4-Y-Z-v2.yaml
-
+- sno-ran-du/sno-ran-du-pg-v4-Y-Z-v3.yaml
+- sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml
```

Comment:
Since this isn't really a progression from v3, but a whole new PG, would it be better to create new sno-ran-full-du directories for the cluster templates and policy templates and start them at v1?